docs/docs/atom-archive/getting-started/sections/atom-basics.md

```diff
@@ -0,0 +1,203 @@
+
+##### Soft Wrap
+
+You can use the Settings View to specify your whitespace and wrapping preferences.
+
+![Whitespace and wrapping preferences settings](@images/atom/settings-wrap.png)
+
+Enabling "Soft Tabs" will insert spaces instead of actual tab characters when you press the <kbd class="platform-all">Tab</kbd> key and the "Tab Length" setting specifies how many spaces to insert when you do so, or how many spaces are used to represent a tab if "Soft Tabs" is disabled.
+
+The "Soft Wrap" option will wrap lines that are too long to fit in your current window. If soft wrapping is disabled, the lines will simply run off the side of the screen and you will have to scroll the window to see the rest of the content. If "Soft Wrap At Preferred Line Length" is toggled, the lines will wrap at 80 characters instead of the end of the screen. You can also change the default line length to a value other than 80 on this screen.
+
+In [Basic Customization](/sections/basic-customization/) we will see how to set different wrap preferences for different types of files (for example, if you want to wrap Markdown files but not other files).
+
+#### Opening, Modifying, and Saving Files
+
+Now that your editor is looking and acting how you want, let's start opening up and editing files. This is a text editor after all, right?
+
+##### Opening a File
+
+There are several ways to open a file in Atom. You can do it by choosing *File > Open* from the menu bar or by pressing <kbd class="platform-mac">Cmd+O</kbd><kbd class="platform-windows platform-linux">Ctrl+O</kbd> to choose a file from the standard dialog.
+
+![Open file by dialog](../../images/open-file.png "Open file by dialog")
+
+This is useful for opening a file that is not contained in the project you're currently in (more on that next), or if you're starting from a new window for some reason.
+
+Another way to open a file in Atom is from the command line using the `atom` command. <span class="platform-mac">The Atom menu bar has a command named "Install Shell Commands" which installs the `atom` and `apm` commands [if Atom wasn't able to install them itself](/getting-started/sections/installing-atom/#installing-atom-on-mac).</span><span class="platform-windows platform-linux">The `atom` and `apm` commands are installed automatically as a part of Atom's [installation process](/getting-started/sections/installing-atom/).</span>
+
+You can run the `atom` command with one or more file paths to open up those files in Atom.
+
+``` command-line
+$ atom --help
+> Atom Editor v1.8.0
+
+> Usage: atom [options] [path ...]
+
+> One or more paths to files or folders may be specified. If there is an
+> existing Atom window that contains all of the given folders, the paths
+> will be opened in that window. Otherwise, they will be opened in a new
+> window.
+
+> ...
+```
+
+This is a great tool if you're used to the terminal or you work from the terminal a lot. Just fire off `atom [files]` and you're ready to start editing.
```

Comment:
So far I have ignored the style of writing, but this one jumped into my eye: "Just fire off" sounds a bit too lax to me.